src/score_gen/score.rs

```diff
@@ -455,35 +468,37 @@ mod tests {
         let pub_params =
             PublicParameters::setup(1 << 17, &mut rand::thread_rng())?;
         let (ck, vk) = pub_params.trim(1 << 16)?;
-        let mut composer = StandardComposer::new();
-        let mut transcript = Transcript::new(b"TEST");
-
-        let res = single_complex_range_proof(
-            &mut composer,
-            Scalar::from(2u64).pow(&[127u64, 0, 0, 0]),
-            Scalar::from(2u64).pow(&[128u64, 0, 0, 0]) - Scalar::one(),
-        )?;
-        // Constraint res to be true, since the range should hold.
-        composer.constrain_to_constant(res, Scalar::one(), Scalar::zero());
-        // Since we don't use all of the wires, we set some dummy constraints to
-        // avoid Committing to zero polynomials.
-        composer.add_dummy_constraints();
-        let prep_circ = composer.preprocess(
-            &ck,
-            &mut transcript,
-            &EvaluationDomain::new(270).expect("Evaluation Domain instance"),
-        );
 
-        let proof = composer.prove(&ck, &prep_circ, &mut transcript.clone());
-        // This should pass since the range_proof holds.
-        assert!(proof.verify(
-            &prep_circ,
-            &mut transcript,
-            &vk,
-            &composer.public_inputs()
-        ));
-
-        Ok(())
+        let complex_rangeproof_gadget =
+            |composer: &mut StandardComposer| -> Result<(), Error> {
+                let res = single_complex_range_proof(
+                    composer,
+                    BlsScalar::from(2u64).pow(&[127u64, 0, 0, 0]),
+                    BlsScalar::from(2u64).pow(&[128u64, 0, 0, 0])
```

Comment:
I'm still thinking that it would more efficient / readable having `BlsScalar::pow_of_2(128)` :) But i's a really, really, low priority thing; it's just "hurt" every time I see so much characters just for that.

Reply:
The issue for that is already opened.